astilectron.go

```diff
@@ -44,6 +44,11 @@ const (
 	EventNameAppTooManyAccept = "app.too.many.accept"
 )
 
+// Unix socket path
+const (
+	UNIX_SOCKET_PATH = "/tmp/astilectron.sock"
```

Comment:
I'd rather not store the socket file outside the `astilectron` folder. I'll describe where it should be in a comment below. You can delete this constant.